src/main/scala/org/opennetworkinsight/LDAWrapper.scala

```diff
@@ -24,6 +24,7 @@ object LDAWrapper {
   =
   {
 
+    println("Proxy topic analysis: gathering words")
```

Comment:
I agree with adding descriptive log like "Topic analysis: gathering words" but I don't think we should use Proxy or any data source name in this file.